xbmc/addons/Scraper.cpp

```diff
@@ -36,6 +36,7 @@
 
 #include <algorithm>
 #include <sstream>
+#include <regex>
```

Comment:
> Included header files have to be sorted (case sensitive) alphabetically to prevent duplicates and allow better overview, with an empty line clearly separating sections.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers